marquez_client/file_backend.py

```diff
@@ -55,7 +59,9 @@ def post(self, path, headers, payload=None):
         if payload:
             post_details['payload'] = payload
 
-        post_json = json.dumps(post_details)
+        post_json = jsonpickle.encode(
+            value=post_details,
+            fail_safe=Utils.handle_fail)
```

Comment:
jsondumps worked in this context. We don't want a resulting json that is dependent on python implementations. It is supposed to be the same on the java and python client